carla_manual_control/src/carla_manual_control/carla_manual_control.py

```diff
@@ -54,14 +58,17 @@
     from tf2_ros import ConnectivityException
     from tf2_ros import ExtrapolationException
     import tf2_ros
-    from rclpy.qos import QoSProfile
+    from rclpy.qos import QoSProfile, QoSDurabilityPolicy
     from threading import Thread, Lock, Event
-    # from builtin_interfaces.msg import Time
+    from builtin_interfaces.msg import Time
     from rosgraph_msgs.msg import Clock
-
-    sys.path.append(os.getcwd() + '/install/ros_compatibility/lib/python3.6/site-packages/src/ros_compatibility')
+    import sys
+    sys.path.append(os.getcwd() +
```

Comment:
Fix importing

Reply:
What are your proposal, can you implement it?

Reply:
Yes, pushed the changes to ros_compatibility branch. We should be able to import now as `from ros_compatibility import *` without any hastle.